internal/app/dispatcher/providers/azurebatch.go

```diff
@@ -455,7 +424,7 @@ func getLogsForTask(ctx context.Context, fileClient *batch.FileClient, t *batch.
 	sb := strings.Builder{}
 	//Log the details from the running task
 	logs, err := getLogFileContent(ctx, t, fileClient, jobID, "stdout.txt")
-	sb.WriteString("\n\n ------ Batch logs: stdout ------ \n\n") //nolint: errcheck
+	sb.WriteString("\n\n ------ [Debug] Batch logs: stdout ------ \n\n") //nolint: errcheck
```

Comment:
The `[DEBUG]` heading does not necessarily match the error level (warnings are used) which might be confusing.

Reply:
Done